content/docs/hello-world.md

```diff
@@ -6,7 +6,7 @@ prev: cdn-links.html
 next: introducing-jsx.html
 ---
 
-The smallest React example looks like this:
+Một ví dụ nhỏ nhất của React trông như thế này:
```

Comment:
`Một ví dụ đơn giản` sẽ gần gũi hơn chị nhỉ?

Reply:
Phần này chị có update gì thêm k?

Reply:
Chị quên update chỗ này đó. Mới commit lên lại nà.

Reply:
Thank you em!